src/OpenTelemetry.Exporter.Prometheus.AspNetCore/README.md

```diff
@@ -65,6 +65,8 @@ dotnet add package --prerelease OpenTelemetry.Exporter.Prometheus.AspNetCore
             context => context.Request.Path == "/internal/metrics"
                 && context.Connection.LocalPort == 5067);
     ```
+    
+    This can be used in combination with [configuring multiple ports on the ASP.NET application](https://learn.microsoft.com/en-us/aspnet/core/fundamentals/servers/kestrel/endpoints?view=aspnetcore-6.0) to expose the scraping endpoint on a different port.
```

Comment:
```suggestion
    This can be used in combination with [configuring multiple ports on the ASP.NET Core application](https://learn.microsoft.com/en-us/aspnet/core/fundamentals/servers/kestrel/endpoints?view=aspnetcore-6.0) to expose the scraping endpoint on a different port.
```

Reply:
Thanks, pushed an update.